packages/shared/src/lib/core/transactions/actions/fetchAndPersistTransactionsForAccounts.ts

```diff
@@ -119,6 +125,21 @@ async function fetchBlockscoutTransactionsForAccount(
     return transactions
 }
 
+async function fetchNovesTransactionsForAccount(
+    account: IAccountState,
+    network: IEvmNetwork
+): Promise<NovesTxsResponse | undefined> {
+    const address = getAddressFromAccountForNetwork(account, network.id)
+    const novesChain = SUPPORTED_NETWORK_ID_TO_NOVES_CHAIN[network.id]
+    if (!address || !novesChain) {
+        return undefined
+    }
+
+    const novesApi = new NovesApi()
+    const transactions = await novesApi.translate.getTransactionsFromAddress(address, novesChain)
```

Comment:
Just looked in and we are not using the pagination for this API call yet, and additionally we will be pulling in transactions from the current time, until the previous time that we pulled in history. So we need to store the time of which we last pulled in history so we do not repeat api calls from previously.